src/Microsoft.Data.SqlClient/netcore/src/Common/src/Interop/Windows/sspicli/SSPIWrapper.cs

```diff
@@ -406,8 +406,7 @@ public static SafeFreeContextBufferChannelBinding QueryContextChannelBinding(SSP
 
         public static object QueryContextAttributes(SSPIInterface secModule, SafeDeleteContext securityContext, Interop.SspiCli.ContextAttribute contextAttribute)
         {
-            int errorCode;
-            return QueryContextAttributes(secModule, securityContext, contextAttribute, out errorCode);
+            return QueryContextAttributes(secModule, securityContext, contextAttribute, out int _);
```

Comment:
Should also be able to leave out the type when doing discards:

```suggestion
            return QueryContextAttributes(secModule, securityContext, contextAttribute, out _);
```

Reply:
This is true. I don't know why the tool I was using preferred to add the type when it wasn't necessary 🤔